app/plugins/basic-authentication.md

```diff
@@ -53,7 +53,7 @@ blacklist groups of users.
 form parameter                             | default | description
 ---                                        | ---     | ---
 `name`                                     |         | The name of the plugin to use, in this case: `basic-auth`
-`config.hide_credentials`<br>*optional*    | `false` | An optional boolean value telling the plugin to hide the credential to the upstream API server. It will be removed by Kong before proxying the request
+`config.hide_credentials`<br>*optional*    | `false` | An optional boolean value telling the plugin to hide the credentials from the upstream API service. If true, credentials will be removed by Kong before proxying the request, implying that a direct request of the upstream does not require basic auth. The auth burden is therefore moved into the plugin. If the upstream still requires basic auth, set `config.hide_credentials=false`
```

Comment:
> the upstream API service

This is confusing. We make a particular effort in these docs to use the same terminology all the time (at least, when we catch such inconsistencies...).

We usually refer to the upstream as "the upstream service", or "your upstream service". 

> The auth burden is therefore moved into the plugin. If the upstream still requires basic auth, set `config.hide_credentials=false`

This part is confusing and seems unnecessary? The whole sentence is also really long.

How about:
```
An optional boolean value telling the plugin to hide the credentials from
the upstream service. If `true`, the plugin will strip the credentials from the
request (i.e., the `Authorization` header) before proxying it.
```

The fact that the "burden" is moved from the upstream service to Kong is... obvious? It is the point of this plugin.

Reply:
Agreed. Fewer, more accurate words more clear. Thank you.